catalog/opensearch/catalog-opensearch-source/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -51,6 +51,17 @@
             type="Boolean" default="true"
             description="Converts Polygon and Point-Radius searches to a Bounding Box for compatability with older interfaces. Generated bounding box is a very rough representation of the input geometry."/>
 
+        <AD description="Disable CN check for the server certificate. This should only be used when testing."
```

Comment:
I've noticed that DDF will present a "Insecure Defaults Found" message in the admin screen. Can that message be automatically triggered when this is set to `true`?